xlsxwriter/worksheet.py

```diff
@@ -428,15 +428,15 @@ def write(self, row, col, *args):
             return self.write_datetime(row, col, *args)
 
         # resort to isinstance for developers who have subclassed a primitive
+        # Write number types.
```

Comment:
This is a good change. `write_string()` should probably move to the second position as well.